src/navigation.js

```diff
@@ -6,7 +6,7 @@ const navigation = [
     subNavigation: [
       {
         id: 'tagYourProject',
-        header: 'Tag Your Project',
```

Comment:
May need to change the cypress test because it's checking for 'Tag Your Project' rather than the newly modified string 'Add Your Project'